utils.py

```diff
@@ -82,15 +85,11 @@ def wrapper_validate(*args, **kwargs):
                         "error_schema": schema_errors.schema,
                         "error_schema_path": list(schema_errors.schema_path),
                     }
-                    log.debug("error response: %s", error_log)
-                    error_response = f"{schema_errors.message} for field"
+                    log.warn("Invalid request (API schema): %s", error_log)
```

Comment:
for the operation and troubleshooting, it is quite useful to also have this in the logs (especially once AMPATH connectors start using Kytos and submitting requests). Thus, I've changed from debug to warning.

Reply:
@italovalcy that's a very great and important point for prod. As it is though, I think this can have additional implications:

- Currently, we've been also using the warning level in the code base for things can cause oddities but can eventually recover, for example the retries with `tenacity` that we have for the DB controllers. So, I'm not sure if we should categorize bad requests from user side in the same category. We have powerful querying capabilities with ES and Kibana, but still this can generate a bit of noise on other warnings. Maybe this a new point that we need to refine and document/categorize better.

- With the message content we don't also have the entire context of the request, so we might not be able to figure out which client has sent this, even though the error message is crystal clear:

```
2022-07-22 17:59:55,140 - WARNING [kytos.napps.kytos/mef_eline] [utils.py:88:wrapper_validate] (Thread-228) Invalid request (API schema): {'error_message': '1 is not of type string', 'error_validator': 'type', 'error_validator_value': 'string', 'error_path': ['uni_a', 'interface_id'], 'error_schema': {'type': 'string', 'minLength': 25, 'maxLength': 30, 'pattern': '^([
0-9A-Fa-f]{2}[:-]){8}(\\d{1,5})$', 'nullable': False}, 'error_schema_path': ['properties', 'uni_a', 'properties', 'interface_id', 'type']}
```

On Elastic APM, if we expose the body capture we can also filter for the status code and see the request body too, although on the free version we won't alerting capability there (so that won't allow us to proactively react to this, but if a client complains it's not hard to find), maybe on nginx we could have the logs configured and index them on ES:

![2022-07-22-181335_1643x333_scrot](https://user-images.githubusercontent.com/1010796/180569412-3ddf3802-948b-4f0c-aed7-c7628cea55ed.png)

![2022-07-22-181318_1641x170_scrot](https://user-images.githubusercontent.com/1010796/180569420-63f42a0c-bc35-4cc5-a288-bfa81203841f.png)

Should we should map a higher level requirement on which kind of requests visibility we need here, and then sort out the solution? Thanks for looking into this.

Reply:
Thank you for your suggestions, @viniarck. I've rolled back to debug level (commit a38fc73) until we have a better solution!

Reply:
Sounds good, @italovalcy, and once again, thanks for looking into this.